src/Blocks/components/load-more/assets/index.js

```diff
@@ -0,0 +1,33 @@
+import domReady from '@wordpress/dom-ready';
```

Comment:
all js files must be written using dynamic Import for optimisations purposes.
Please refer to the carousel assets or any other assets in the 8shift-frontend-libs or solplanet project

Reply:
That's exactly how it is in the [8shift-frontend libs Carousel block](https://github.com/infinum/eightshift-frontend-libs/blob/develop/blocks/init/src/Blocks/custom/carousel/assets/index.js) 😁

Reply:
Is this the correct way to do it?

```js
async function load() { 
	let domReady = await import('@wordpress/dom-ready');

	domReady.default(() => {
	    // ...
	});
}

load();
```

Reply:
Why did you use async await here? 

this is allready dynamic import
https://github.com/infinum/eightshift-frontend-libs/blob/c84e4dd0af2ff05cd2cd7bcc62e5e0097c39ed55/blocks/init/src/Blocks/custom/carousel/assets/index.js#L10

read about this:
https://medium.com/front-end-weekly/webpack-and-dynamic-imports-doing-it-right-72549ff49234

https://medium.com/front-end-weekly/webpack-and-dynamic-imports-doing-it-right-72549ff49234

Reply:
I use async/await because

```js
let bar = await foo();
// use bar
```
is the same as
```js
foo().then((bar) => { 
    // use bar
});
```

I like async/await more because it works the same, but removes the unnecessary indentation and brackets.
(same with `x.forEach((y) => {})` and `for (let y of x) {}` 😄 )

But if `.then()` is how it's done everywhere, I'll change it to be consistent with everything.

Reply:
BTW in the Carousel example, `domReady` isn't being dynamically imported, right?

Reply:
We are using webpack dynamic import because then webpack handles when the file is actually loaded in the DOM witch reduces the JS file sizes.

Please use carousel as reference. 

No domReady is not dynamically imported

Reply:
If I understood you correctly, I should move most of the logic into a separate file and then in index.js dynamically import it?

Should `domReady` be dynamically imported or is it ok if it stays like it is in Carousel?

Reply:
Please just check this other examples and make it like this:

https://github.com/infinum/wordpress-solplanet/tree/master/wp-content/themes/solplanet/src/Blocks/custom/carousel/assets